iXBRLViewerPlugin/viewer/src/less/viewer.less

```diff
@@ -41,9 +41,21 @@ span {
     outline-offset: 1px;
   }
 
-  &.ixbrl-element:hover:not(.ixbrl-selected) {
-    outline: dashed 2px @primary-focus;
-    outline-offset: 1px;
+  &.ixbrl-element {
+    display: inline-block;
+
+    & > [contextref] {
+      display: flex;
+    }
+
+    & > ix\:continuation {
```

Comment:
I'm nervous that this requires `xmlns:ix`, but someone might choose to generate without (e.g., edit an existing iXBRL doc to use `xmlns:ixbrl` and `ixbrl:` throughout).  I think the JS adds a class, so just check `.ixbrl-continuation` if that's required?

Reply:
Good point, `.ixbrl-continuation` can be used to the same affect.